package.json

```diff
@@ -53,7 +53,7 @@
     "stylelint-config-prettier": "3.2.0",
     "tslint": "5.10.0",
     "tslint-config-prettier": "1.12.0",
-    "tslint-sonarts": "1.6.0",
+    "tslint-sonarts": "1.7.0",
```

Comment:
This upgrade breaks the build since there are some bugs in SonarTS 1.7

We will revert this to v1.6 in another PR